pkg/sfu/downtrack.go

```diff
@@ -1506,12 +1506,16 @@ func (d *DownTrack) handleRTCP(bytes []byte) {
 	for _, pkt := range pkts {
 		switch p := pkt.(type) {
 		case *rtcp.PictureLossIndication:
-			numPLIs++
-			sendPliOnce()
+			if p.MediaSSRC == d.ssrc {
```

Comment:
Was thinking if we should check `SenderSSRC` also, some thing like
```
if p.MediaSSRC == d.ssrc || p.SenderSSRC == d.ssrc {
````
just to be defensive.

Reply:
Not sure if the senderSSRC can be set to another stream's ssrc, will confirm it

Reply:
Another option is to do `if p.MediaSSRC == d.ssrc || p.MediaSSRC == 0`. I am just concerned about some client somewhere not setting it properly.

Reply:
I think it is safe to only check MediaSSRC, our clients don't have the incorrect media ssrc issue. If there is a client has such case it should be fixed by client as it doesn't comply the RFC. Will add a warn log if a packet dispatched to the downtrack while neither MediaSSRC nor SenderSSRC is equal to its ssrc.

Reply:
Sounds good. Thank you @cnderrauber .

Reply:
chrome/libwebrtc use a constant sender ssrc `1` for all rtcp fb messages.

Reply:
🤦